imports/ui/components/form-controls.mui-styles.js

```diff
@@ -14,3 +14,15 @@ export const textInputUnderlineFocusStyle = {
 export const selectInputIconStyle = {
   fill: 'var(--mid-gray)'
 }
+
+export const whiteTextInputStyle = {
+  backgroundColor: '#fff',
+  borderRadius: '20px',
```

Comment:
The border radius of the original input made it have the shape of a "pill", but it was effectively 16px and not 20

Reply:
Will change to 16px. I initially saw the white pill chatbox included br-pill class (border-radius: 9999px in main.css)